packages/Tag/stories/Examples/Variations.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable react/jsx-wrap-multilines */
+/* eslint-disable react/no-array-index-key */
+import React from "react";
+import L10n from "@paprika/l10n";
+import CalendarIcon from "@paprika/icon/lib/Calendar";
+import { action } from "@storybook/addon-actions";
+import ExclamationCircle from "@paprika/icon/lib/ExclamationCircle";
+import tokens from "@paprika/tokens";
+import Avatar from "../../../Avatar/src";
+import Tag, { Tags } from "../../src";
+
+const handleRemove = () => {
+  action("Remove event")();
+};
+
+const handleClick = () => {
+  action("Click event")();
+};
+
+export default function Variations() {
+  return (
+    <L10n locale="en">
+      <div style={{ padding: "20px" }}>
+        Variations
+        {[{ tagSize: "medium", avatarSize: "small" }, { tagSize: "large", avatarSize: "medium" }].map(permutation => (
```

Comment:
I like this idea of mapping the permutations! Do you think it would be more readable to extract the `permutations` array into a variable declared at the top?

Reply:
Personally I find it quite readable when it is shown at the point of mapping in the file, rather than having to hunt around for a variable. Perhaps if there were many more options I would agree it belongs in a set variable but i think is ok here